### ISSUE TITLE
stats does not have any effect

### ISSUE THREAD

**Issue by tommedema** [serverless-heaven/serverless-webpack#283]
# This is a Bug Report

## Description

For bug reports:
* What went wrong?

I supplied `stats: error-only` in webpack.config.js but still received verbose output:

```
> serverless deploy --verbose --stage prod --aws-s3-accelerate

Serverless: Using multi-compile (individual packaging)
Serverless: Bundling with Webpack...
Time: 743ms
                         Asset     Size  Chunks             Chunk Names
scripts/request-certificate.js  3.06 kB       0  [emitted]  scripts/request-certificate
   [0] ./scripts/request-certificate.js 405 bytes {0} [built]
   [1] external "cfn-response" 42 bytes {0} [not cacheable]
Time: 808ms
                  Asset     Size  Chunks             Chunk Names
src/functions/create.js  6.78 kB       0  [emitted]  src/functions/create
   [0] ./src/functions/create.js 2.38 kB {0} [built]
   [1] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "babel-runtime/helpers/asyncToGenerator" 42 bytes {0} [not cacheable]
   [3] external "uuid" 42 bytes {0} [not cacheable]
   [4] ./src/functions/libs/dynamodb-lib.js 427 bytes {0} [built]
   [5] external "aws-sdk" 42 bytes {0} [not cacheable]
   [6] ./src/functions/libs/response-lib.js 799 bytes {0} [built]
   [7] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
Time: 807ms
               Asset     Size  Chunks             Chunk Names
src/functions/get.js  6.66 kB       0  [emitted]  src/functions/get
   [0] ./src/functions/get.js 2.35 kB {0} [built]
   [1] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "babel-runtime/helpers/asyncToGenerator" 42 bytes {0} [not cacheable]
   [3] ./src/functions/libs/dynamodb-lib.js 427 bytes {0} [built]
   [4] external "aws-sdk" 42 bytes {0} [not cacheable]
   [5] ./src/functions/libs/response-lib.js 799 bytes {0} [built]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
Time: 807ms
                Asset     Size  Chunks             Chunk Names
src/functions/list.js  6.51 kB       0  [emitted]  src/functions/list
   [0] ./src/functions/list.js 2.21 kB {0} [built]
   [1] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "babel-runtime/helpers/asyncToGenerator" 42 bytes {0} [not cacheable]
   [3] ./src/functions/libs/dynamodb-lib.js 427 bytes {0} [built]
   [4] external "aws-sdk" 42 bytes {0} [not cacheable]
   [5] ./src/functions/libs/response-lib.js 799 bytes {0} [built]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
Time: 807ms
                  Asset     Size  Chunks             Chunk Names
src/functions/update.js  6.84 kB       0  [emitted]  src/functions/update
   [0] ./src/functions/update.js 2.53 kB {0} [built]
   [1] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "babel-runtime/helpers/asyncToGenerator" 42 bytes {0} [not cacheable]
   [3] ./src/functions/libs/dynamodb-lib.js 427 bytes {0} [built]
   [4] external "aws-sdk" 42 bytes {0} [not cacheable]
   [5] ./src/functions/libs/response-lib.js 799 bytes {0} [built]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
Time: 807ms
                  Asset     Size  Chunks             Chunk Names
src/functions/delete.js  6.47 kB       0  [emitted]  src/functions/delete
   [0] ./src/functions/delete.js 2.17 kB {0} [built]
   [1] external "babel-runtime/regenerator" 42 bytes {0} [not cacheable]
   [2] external "babel-runtime/helpers/asyncToGenerator" 42 bytes {0} [not cacheable]
   [3] ./src/functions/libs/dynamodb-lib.js 427 bytes {0} [built]
   [4] external "aws-sdk" 42 bytes {0} [not cacheable]
   [5] ./src/functions/libs/response-lib.js 799 bytes {0} [built]
   [6] external "babel-runtime/core-js/json/stringify" 42 bytes {0} [not cacheable]
```

* What did you expect should have happened?

I expected no webpack output since there are no errors.

* What was the config you used?

```
const slsw = require('serverless-webpack')
const nodeExternals = require('webpack-node-externals')

module.exports = {
  entry: slsw.lib.entries,
  target: 'node',
  stats: 'errors-only',
  externals: [nodeExternals()],
  module: {
    rules: [
      {
        test: /\.js$/,
        loader: 'babel-loader',
        include: __dirname,
        exclude: /node_modules/
      }
    ]
  }
}

```

* What stacktrace or error message from your provider did you see?

Similar or dependent issue(s):
* 

## Additional Data

* ***Serverless-Webpack Version you're using***:
4.0.0
* ***Webpack version you're using***:
3.8.1
* ***Serverless Framework Version you're using***:
1.24.1
* ***Operating System***:
OSX
* ***Stack Trace (if available)***:

**Comment by HyperBrain** [comment]
Hi @tommedema ,

this is already fixed in master with #278 and will be released with 4.1.0 soon in a few days.

Thanks for reporting.

**Comment by tommedema** [comment]
Thank you!
